### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by navi86**
Fix tests
Update search path should be done only when path is changed.

### CODE REVIEW

File: postgresql/resource_postgresql_role.go [hunk 1]
```diff
@@ -449,15 +449,15 @@ func resourcePostgreSQLRoleReadImpl(c *Client, d *schema.ResourceData) error {
...
+// In case no such value is present, it returns nil.
 func readSearchPath(roleConfig pq.ByteaArray) []string {
 	for _, v := range roleConfig {
 		config := string(v)
 		if strings.HasPrefix(config, roleSearchPathAttr) {
 			return strings.Split(strings.TrimPrefix(config, roleSearchPathAttr+"="), ", ")
 		}
 	}
-	return []string{"DEFAULT"}
+	return nil
```
Comment (Hjdskes) [inline postgresql/resource_postgresql_role.go:460]: How does one now reset the search path back to default?

**Comment by Hjdskes** [comment]
Thanks for taking the initiative on this! I just have one question, see the inline comment :)

File: postgresql/resource_postgresql_role.go (hunk 1 above)
Comment (navi86) [reply postgresql/resource_postgresql_role.go:460]: when terraform see that need to reset it executes function "alterSerachPath".
Please check, I've tested very late :(
I hope your functionality works as expected.

**Comment by Hjdskes** [comment]
Thanks!

**Pull request merged**